### PR TITLE
Add testing for ColoredFormatter

### PR DESCRIPTION
**Pull request by delta9000**
Checks ANSI escape codes for correct coloring based on the DEFAULT_COLORS variable using the LogCapture class from testfixtures and the ColoredFormatter subclass.  100% coverage on log.py.

#44 
#48

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,6 +6,7 @@ deps =
     pytest
     pytest-cov
     codecov
+	testfixtures
```
Comment (rougeth) [inline tox.ini]: Change `tab` for `spaces`

File: tests/test_cli.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import logging
+import logging, logging.config
 import os
 
 import click
 from click.testing import CliRunner
+from testfixtures import LogCapture
 
 from bottery.cli import cli, debug_option
+from bottery.log import ColoredFormatter,DEFAULT_LOGGING
```
Comment (rougeth) [inline tests/test_cli.py]: Space after `, `

File: tests/test_cli.py [hunk 3]
```diff
@@ -48,3 +50,24 @@ def test_startproject():
...
+
+		
+def test_ColoredFormatter():
+    c = ColoredFormatter()
+    expected = ['DEBUG',
+                '\x1b[34mINFO\x1b[0m',
+                '\x1b[33mWARN\x1b[0m',
+                '\x1b[31mERROR\x1b[0m',
+                '\x1b[30m\x1b[41mCRITICAL\x1b[0m'
+               ]
```
Comment (rougeth) [inline tests/test_cli.py]: pep8 :)
```python
expected = [
    'DEBUG',
    '\x1b[34mINFO\x1b[0m',
    '\x1b[33mWARN\x1b[0m',
    '\x1b[31mERROR\x1b[0m',
    '\x1b[30m\x1b[41mCRITICAL\x1b[0m'
]
```

Comment (delta9000) [reply tests/test_cli.py]: Sorry, a bit new at this.  Should tox be catching these?  I ran it without errors.

File: tox.ini (hunk 1 above)
Comment (delta9000) [reply tox.ini]: fixed in latest commit

File: tests/test_cli.py (hunk 3 above)
Comment (delta9000) [reply tests/test_cli.py]: fixed in latest commit

File: tests/test_cli.py (hunk 2 above)
Comment (delta9000) [reply tests/test_cli.py]: fixed in latest commit

File: tests/test_cli.py [hunk 4]
```diff
@@ -48,3 +51,25 @@ def test_startproject():
...
+    ]
+    logging.config.dictConfig(DEFAULT_LOGGING)
+    with LogCapture(names='bottery') as l:
+        logger = logging.getLogger('bottery')
+        logger.debug('DEBUG')
+        logger.info('INFO')
+        logger.warning('WARN')
+        logger.error('ERROR')
+        logger.critical('CRITICAL')
+        records = [x for x in l.records]
```
Comment (nicoddemus) [inline tests/test_cli.py]: The following will probably produce a better error message:

```python
formatted_records = [c.format(x) for x in l.records]
assert formatted_records == expected
```

**Comment by nicoddemus** [comment]
>I ran the file through a PEP8 checker, found a few other things. Should tox be catching these?

We don't run any PEP8 checks, only flake8, but we could easily add PEP8 checks if we want to.

**Comment by rougeth** [comment]
@leportella just opened a PR to add `flake8` and `isort` on tests.

**Pull request merged**